tests/6.types.out

```diff
@@ -1,6 +1,15 @@
 [translated]
 module main
 
+type Wchar_t = int
```

Comment:
The failing trst seem to miss an `Rtype_t ...` entry here?

Reply:
I'm not sure what to do about that? It's only on macOS.

Reply:
Yeah, it's odd that it only happens on macOS 🤔

Reply:
I don't understand c2v well enough to get why it pulls out a few of these typedefs from the headers and not others (assuming that is what it's doing). Does that look like a bug? Or would it be better to make the "expected output" file able to handle macOS / linux differences?

Reply:
I'm not very deep into the project either. One thing I can think of is that maybe there's differences in what versions of clang is used that maybe produces different AST nodes :man_shrugging:

Reply:
I think output should be the same on all platforms - otherwise we'll end up in "special-case-hell" in the long run. If it happens to be compiler versions maybe there can be implemented exceptions - but in general I think it's important not to have exceptions if it can be avoided

Reply:
Someone who understands c2v better should probably try to fix this. For now, I will try to remove the include which is causing the issue.